web/app/controllers/SysconfigController.php

```diff
@@ -54,6 +54,16 @@
         }
     }
 
+    // EXTERNAL AUTH
+    if ($Request->request->has('extauth_remote_user')) {
```

Comment:
this can be merged into the following codeblock triggered with "updateConfig" and must be removed